solidity/random-datasource/randomExample.sol

```diff
@@ -2,49 +2,48 @@
    Oraclize random-datasource example
```

Comment:
Maybe we should add somewhere in the example itself that this works on _Remix_ only with `injected web3`?

Reply:
Good idea, though I won't add `remix` since it also works in Truffle etc. The real restriction is `live test-nets`. Added!